app/assets/javascripts/snowcrash/modules/issues/table.js.coffee

```diff
@@ -1,13 +1,59 @@
 class IssueTable
+  $table: null
+  selectedColumns: []
+
   constructor: ->
-    $('#issue-table').on('click', '.js-taglink', @tagSelected)
+    @$table       = $('#issue-table table')
+    @$column_menu = $('.dropdown-menu.js-table-columns')
+
+    # -------------------------------------------------------- Load table state
+    @loadColumnState()
+    @showHideColumns()
+
+    # -------------------------------------------------- Install event handlers
+    $('#issue-table').on('click', '.js-taglink', @onTagSelected)
+
     # We're hooking into Rails UJS data-confirm behavior to only fire the Ajax
     # if the user confirmed the deletion
-    $('#issue-table').on('confirm:complete', '#delete-selected', @deleteSelected)
+    $('#issue-table').on('confirm:complete', '#delete-selected', @onDeleteSelected)
+
+    # Handle the showing / hiding of table columns
+    @$column_menu.find('a').on 'click', @onColumnPickerClick
+
+  loadColumnState: =>
+    # TODO: persist this in browser local storage or a cookie
+    @selectedColumns = ['title', 'tags', 'affected']
+    that = this
+
+    @$column_menu.find('a').each ->
+      $link = $(this)
+      if that.selectedColumns.indexOf($link.data('column')) > -1
+        $($link.find('input')).prop('checked', true)
```

Comment:
The other `$()` doesn't do anything here, `$($link.find('input'))` is equivalent to just `$link.find('input')`